designer/server/src/main/scala/pl/touk/nussknacker/ui/process/migrate/RemoteEnvironment.scala

```diff
@@ -103,6 +108,10 @@ class HttpRemoteEnvironment(
     )
   }
 
+  override def close(): Unit = Await.ready(http.shutdownAllConnectionPools(), closeTimeout)
+
+  @unused
```

Comment:
is it needed? the annotation.
BTW, I thought that this annotation was introduced in Scala 2.13. But the cross-compile doesn't complain ... weird.

Reply:
Depends on how you see it. I added it because it's an API that I believe should be available here, but it's unused in Nussknacker. This annotation is meant exactly for this use case.

Reply:
I can remove it if you don't want it here

Reply:
If the method is not used, do we need to have it? If the method can be used somewhere else, eg. in some Nu extension, IMO, the annotation is not needed.

Reply:
Done, annotation removed